libs/core/src/lib/avatar/avatar.component.ts

```diff
@@ -79,25 +79,28 @@ export class AvatarComponent implements OnChanges, OnInit, CssClassBuilder {
     @Input() colorAccent: ColorAccent = null;
 
     /**
-    * @deprecated
-    * Deprecated according to Rename `backgroundImage` to `image`
-    * */
+     * @deprecated
+     * Deprecated according to Rename `backgroundImage` to `image`
+     * */
     /** Background image url. */
     @Input()
     set backgroundImage(value: string) {
         this._setImage(value);
-    };
+    }
 
     /** Background image resource: url or base64. */
     @Input()
     set image(value: string) {
         this._setImage(value);
     }
+    get image(): string {
```

Comment:
I do not understand the purpose of the changes to this file

Reply:
It is needed to make the avatar image accessble here in the template
https://github.com/SAP/fundamental-ngx/pull/3344/files#diff-df9fe7dae64471bbb9f2e6b9e980a247R7